web-client/src/translations/es-ES.json

```diff
@@ -88,8 +88,9 @@
     "login": {
       "title": "Reach4Help",
       "sub_title": "Please Note",
-      "info": "We are only allowing Facebook Users with a verified account 6 months or older for safety. \nWe are working hard to bring more ways to login in the future. \nThank you for your patience.",
-      "facebookButtonLabel": "Continue with Facebook"
+      "info": "We are only allowing Google or Facebook Users with a verified account 6 months or older for safety. \nWe are working hard to bring more ways to login in the future. \nThank you for your patience.",
+      "facebookButtonLabel": "Continue with Facebook",
+      "googleButtonLabel": "Continue with Google"
```

Comment:
These files are updated automatically by the translation integration. Only update `en.json` (as you've already done 👍 ).

Reply:
_Remove these edits_ 😄

Reply:
sounds good, fixed!